TheaterSchedule.DAL/Repositories/UserRepository.cs

```diff
@@ -84,5 +84,23 @@ public void UpdateUser(ApplicationUserModel user)
             UpdateUser.PasswordHash = user.PasswordHash;
             UpdateUser.PhoneNumber = user.PhoneNumber;              
         }
+
+        public async Task UpdatePasswordAsync(ChangePasswordModel model)
+        {
+            var user = await db.Account.FirstAsync(item => item.AccountId == model.Id);
+            user.PasswordHash = model.Password;
+        }
+
+        public async Task UpdateProfileAsync(ChangeProfileModel model)

```

Comment:
use models in service layer, not in db layer

Reply:
ChangePasswordModel and ChangeProfileModel are models from DAL layer, where I used them.
In BLL layer I also have ChangePasswordDTO and ChangeProfileDTO